code/modules/mob/living/carbon/resist.dm

```diff
@@ -2,31 +2,37 @@
 	set name = "Resist"
 	set category = "IC"
 
-	if(!stat && can_click())
-		resist_grab()
-		if(!weakened)
-			process_resist()
+	if(can_resist())
+		next_move = world.time + 20
+		process_resist()
+
+/mob/living/proc/can_resist()
+	//need to allow !canmove, or otherwise neck grabs can't be resisted
+	//so just check weakened instead.
+	if(stat || weakened)
+		return 0
+	if(next_move > world.time)
+		return 0
+	return 1
 
 /mob/living/proc/process_resist()
 	//Getting out of someone's inventory.
 	if(istype(src.loc, /obj/item/weapon/holder))
 		escape_inventory(src.loc)
 		return
 
+	//resisting grabs (as if it helps anyone...)
+	if (!restrained())
+		resist_grab()
```

Comment:
This is probably a better solution to the current method of stopping people from resisting grabs while cuffed on line 86 so you can remove this from resist.dm:
```
/mob/living/carbon/resist_grab()
	return !handcuffed && ..()
```

Reply:
Alright i'll make a separate PR with that and close this one since you already have one for resists @Garen7

Reply:
thanks man